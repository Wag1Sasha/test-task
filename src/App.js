import React,{useState} from 'react'
import { NavBar } from './components/NavBar';
import { FormPage } from './pages/FormPage';
import { PhotosPage } from './pages/PhotosPage';
import {
  BrowserRouter as Router,
  Switch,
  Route,
} from "react-router-dom";



export const App =()=> {

  const [lastTags,setLastTags] = useState([])

  // const sortLastTags =()=>{
  //   if(lastTags.length === 3){
  //      lastTags.splice(1,-1)
  //   }
  // }

  return (
    <>
        <NavBar />
          <Switch>
            <Route exact path="/" component={FormPage} />
            <Route path='/photos/:value' component={PhotosPage}/>
          </Switch>
    </>
  );
}


