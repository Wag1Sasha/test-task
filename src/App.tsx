import React from 'react';
import { Switch, Route } from 'react-router-dom';

import { NavBar } from './components/NavBar';
import { FormPage } from './pages/FormPage';

export const App = () => (
  <>
    <NavBar />
    <Switch>
      <Route exact path="/" component={FormPage} />
    </Switch>
  </>
);
