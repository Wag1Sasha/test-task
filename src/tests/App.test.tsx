import React from 'react';
import { shallow } from 'enzyme';

import { App } from '../App';

describe('should render App component', () => {
  
  it('render App component', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
