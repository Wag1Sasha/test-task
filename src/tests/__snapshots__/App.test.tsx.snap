// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component render App component 1`] = `
<Fragment>
  <NavBar />
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
  </Switch>
</Fragment>
`;
